packages/application-components/src/components/page-not-found/messages.js

```diff
@@ -8,10 +8,6 @@ export default defineMessages({
   paragraph1: {
     id: 'PageNotFound.paragraph1',
     defaultMessage:
-      'The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools {link} if you have any further questions.',
-  },
-  helpDesk: {
-    id: 'PageNotFound.helpDesk',
-    defaultMessage: 'Help Desk',
+      'The item you are looking for may have been deleted, does not exist, or the URL was entered incorrectly. Check the URL and try again.\n\nPlease contact your system administrator or the commercetools <a>Help Desk</a> if you have any further questions.',
```

Comment:
I had to change this as the current implementation was broken (the message link was compiled as `[Object]`)

Reply:
I'm not sure if this might be a breaking change or not, we need to check...